### ISSUE TITLE
[Bug] Pop-Os 24.04 Alpha does not save my output sound settings.

### ISSUE THREAD

**Issue by loranorge** [pop-os/cosmic-epoch#904]
Hi everyone.
The problem I have is that Pop-Os 24.04 Alpha does not save the output sound settings. Every time I turn on the computer I have to go to the output sound settings and first press "Pro audio" and then press "Play hifi quality (hdmi1, hdmi2, hdmi3, mic1, mic2, speaker)". This brings back the sound of the videos and songs. I don't have this problem in other linux distros, so it is not a hardware problem. 
![20240906_170117](https://github.com/user-attachments/assets/70e41c7e-7d2b-4b7a-98e3-67248bc54336)
I get an error when opening videos, and it is the one I explain below. I don't know if this influences the error explained above.
![20240906_164200](https://github.com/user-attachments/assets/de7939bf-ae16-46c7-b961-9b5c766bd3e9)

These are the errors that occur on my Lenovo Ideapad i5-1135G7 laptop, 16GB of RAM, 512 GB and Intel Iris Xe integrated graphics card. If you need any further information, please let me know. By the way, can you please tell me what can be done in these cases?

Greetings from Spain.

**Comment by mmstick** [comment]
gnome-videos will show that error because we don't use gnome's tracker service in COSMIC.

**Comment by loranorge** [comment]
Thank you very much for your reply, but I use the DeaDBeeF-Player application to listen to music and it is not from Gnome and I get the error. How can I solve this error?

**Comment by ahoneybun** [comment]
It uses a GNOME backend service that is not shipped with COSMIC which is the issue. Installing this should do it:

sudo apt install tracker tracker-extract tracker-miner-fs

**Comment by loranorge** [comment]
Thank you very much for answering me, but the solution you offered me did not work. It still does not save the output sound configuration. I installed the tracker program from the terminal and it did nothing. I had to uninstall Pop-Os 24.04 Alpha and install it from scratch, because I could not uninstall the program. Anyway, I am back to where I was before. The program says it has been installed, but it does not appear anywhere.

**Comment by mmstick** [comment]
That is for the error you made of a screenshot of from gnome-videos. Not sound configurations.

**Comment by loranorge** [comment]
Good morning, all the errors have been solved, I'll tell you. First error: The window with the error that appeared when playing the videos has disappeared when installing Tracker. 2 and 3 errors: The sound configuration is now saved and the browser no longer pops up every time I listened to music. These errors have been solved by installing Pop-Os 24.04 Alfa on my desktop computer on an internal SSD drive. Conclusion: Installing Pop-Os 24.04 Alfa on an external SSD drive will give errors in the music player and in the output sound configuration. The ideal thing is to install it on an internal SSD drive. The only thing I have not checked is installing it together with Windows 11, now it is impossible, because the Pop-Os 24.04 Alfa installer partition is broken. I hope that in the Alfa 2 version, it will be operational and I can install it.
